api/ccapi/include/layer.h

```diff
@@ -359,6 +359,17 @@ Addition(const std::vector<std::string> &properties = {}) {
   return createLayer(LayerType::LAYER_ADDITION, properties);
 }
 
+#ifdef ENABLE_OPENCL
+/**
+ * @brief Helper function to create Addition layer for GPU
+ */
+inline std::unique_ptr<Layer>
+AdditionCL(const std::vector<std::string> &properties = {},
```

Comment:
Don't need to create `AdditionCL` separately. Existing `Addition` API can be modified with `LayerComputeEngine`. Since both of them use the same `LayerType::LAYER_ADDITION`.

Reply:
Updated in the new commit.
Thanks.